Cargo.toml

```diff
@@ -12,4 +12,4 @@ repository = "https://github.com/Barugon/egui_file"
 version = "0.12.0"
 
 [dependencies.egui]
-version = "0.24"
+version = "0.24.1"
```

Comment:
This should stay "0.24". Patch releases are picked up when you run `cargo update` (or just delete `Cargo.lock`).